src/five/formlib/tests/formlib.txt

```diff
@@ -45,21 +45,30 @@ Let's 'manually' create a Content instance and add it to the folder:
   >>> obj = content.Content('content_1', 'Title', [])
   >>> folder._setObject('content_1', obj)
   'content_1'
-  >>> print folder.content_1.title
+  >>> print (folder.content_1.title)
   Title
 
 Now we can edit this content object, e.g. changing the title. Formlib,
-like the traditional AddView and EditView, supports unicode:
+like the traditional AddView and EditView, supports unicode.
+But we need some hackery for Python 2/3 compatibility. For Python 2,
+the test framework expects utf-8 encoded binary strings; for Python 3,
+it expects text. The function below converts as necessary.
+
+  >>> def to_input(v):
+  ...     if hasattr("", "decode"):
```

Comment:
In Zope we use the `six` library to differentiate between the Python versions.
See [this example](https://github.com/zopefoundation/Zope/blob/e6a85acc02b12d349db96b487027b138fb897cbc/src/Zope2/Startup/tests/test_starter.py#L58-L61).
As `Zope` is already a dependency of `five.formlib` the `six` package is a very light weight extra dependency.

There is another copy of this function, which could be changed, too.

Reply:
@icemac I need help.
After changing `.travis.yml`, the CI lets all checks fail with errors that have nothing to do with `five.formlib`, e.g.
```
#29.1 failed
...
    from AccessControl.ZopeGuards import guarded_getitem  # NOQA
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/AccessControl/ZopeGuards.py", line 25, in <module>
    from RestrictedPython.Guards import guarded_iter_unpack_sequence
ImportError: cannot import name guarded_iter_unpack_sequence
```
or
```
#29.2 errored
...
Failed building wheel for Persistence
...
Command "/home/travis/virtualenv/python3.4.6/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-qxilafba/Persistence/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-iu5sfash/install-record.txt --single-version-externally-managed --compile --install-headers /home/travis/virtualenv/python3.4.6/include/site/python3.4/Persistence" failed with error code 1 in /tmp/pip-install-qxilafba/Persistence/
The command "pip install -U -e "."" failed and exited with 1 during .
```
Can you please have a look and tell me, what went wrong.